WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -88,6 +102,20 @@ final class ShippingLabelFormViewModel {
         return [Section(rows: rows)]
     }
 
+    // Return the body of the Package Details cell
+    //
+    func getPackageDetailsBody() -> String {
+        guard let packagesResponse = packagesResponse,
+              let selectedPackageID = selectedPackageID,
+              let totalPackageWeight = totalPackageWeight else {
+            return Localization.packageDetailsPlaceholder
+        }
+
+        let packageTitle = searchCustomPackage(id: selectedPackageID)?.title ?? searchPredefinedPackage(id: selectedPackageID)?.title ?? ""
+        let packageWeight = totalPackageWeight + " " + packagesResponse.storeOptions.dimensionUnit
```

Comment:
Should be weight unit instead of dimension

```suggestion
        let packageWeight = totalPackageWeight + " " + packagesResponse.storeOptions.weightUnit
```

Reply:
Can we use `MeasurementFormatter` here? I see there is even [WeightFormatter](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/Tools/WeightFormatter.swift) wrapper ready.

Reply:
It totally makes sense 💯 updated here: 010ccb50917104e22bf98dd72572ce3ba69becac